airbyte-integrations/connectors/destination-milvus/destination_milvus/indexer.py

```diff
@@ -35,7 +34,7 @@ def _connect(self):
             token=self.config.auth.token if self.config.auth.mode == "token" else "",
         )
 
-    def _create_client(self):
+    def _connect_with_timeout(self):
```

Comment:
Here, and below for `_create_client()`, I'm noticing that lack of an `@override` decorator makes it tricky for the reader and author to effectively communicate if the named methods are internal to this class or if they are implementing/overriding base class behavior.

For readability and type checking, what do you think of adding an `@override` in our CDK implementation, as described in [PEP 698](https://peps.python.org/pep-0698/)? While not yet supported natively in Python, there's apparently an [`overrides`](https://pypi.org/project/overrides/) library that can support this, including an `EnforceOverrides` mixin that would make override declarations required in subclasses of a CDK-declared class. (Downside is that enforcing this will break all downstream connectors.)

Note: I don't think there's a path for us to implement here, nor would it be a blocker on the current PR. Would require further discussion and would likely be a longer-term investment if we decide to go this direction.

Reply:
Sounds like an interesting idea, could you open an issue for this and tag the extensibility team?